### PR TITLE
Add `{gt}` to `Suggests`

### PR DESCRIPTION
**Pull request by IndrajeetPatil**
Since it's needed by one of the tests.

### CODE REVIEW

File: DESCRIPTION [hunk 1]
```diff
@@ -1,7 +1,7 @@
 Type: Package
 Package: effectsize
 Title: Indices of Effect Size
-Version: 0.8.2.00005
+Version: 0.8.2.6
```
Comment (IndrajeetPatil) [inline DESCRIPTION:4]: To be consistent with versioning adopted in the rest of the ecosystem.

**Comment by mattansb** [comment]
Why is this needed? 🤔

**Comment by IndrajeetPatil** [comment]
https://github.com/easystats/effectsize/blob/2eace7521e94e2faab9cdb5b2a3dab6f0dafe3fc/tests/testthat/test-print.R#L46

**Comment by mattansb** [comment]
Merd...

**Pull request merged**